.github/workflows/release.yaml

```diff
@@ -166,8 +164,21 @@ jobs:
       - name: Build  🔧
         env:
           FLAVOR: ${{ matrix.flavor }}
+          FLAVOR_RELEASE: ${{ matrix.flavorRelease }}
+          FAMILY: ${{ matrix.family }}
+          ARTIFACT: ${{ matrix.artifact }}
+          VARIANT: ${{ matrix.variant }}
+          MODEL: ${{ matrix.model }}
+          BASE_IMAGE: ${{ matrix.baseImage }}
```

Comment:
I guess these can be removed (also the one above)

Reply:
Correct! Fixed.